source/ed/build_helper.cpp

```diff
@@ -69,6 +70,7 @@ VJ::VJ(builder& b,
     start_time(start_time),
     end_time(end_time),
     headway_secs(headway_secs),
+    _vj_type(vj_type),
```

Comment:
Let's save "`_`private_attributes" for python :pray: 
(if you can remove the one below it's nice too)

Reply:
There are different schools :smile:. It can be applicable for c ++ too. 
But in the scope of this class, it is not the main convention. So, I change that

Reply:
but i'll not clean all internal vars with this convention.

Reply:
That's fine, it would be nice to do it someday though, as it's not the convention in kraken.